Sources/Redis+Geo.swift

```diff
@@ -0,0 +1,324 @@
+    /// - parameter members: List of members from which to get their Geohash
+    ///                      strings.
+    /// - parameter callback: The callback function.
+    /// - parameter result: An array where each element is the Geohash
+    ///                     corresponding to each member name passed as argument
+    ///                     to the command.
+    /// - parameter error: Non-nil if error occurred.
+    public func geohashArrayOfMembers(key: String, members: [String], callback: (_ result: [RedisString?]?, _ error: NSError?) -> Void) {
+        var command = ["GEOHASH", key]
+        for member in members {
+            command.append(member)
+        }
+        issueCommandInArray(command) { (response) in
+            redisStringArrayResponseHandler(response, callback: callback)
+        }
+    }
+    
+    /// Return the positions (longitude,latitude) of all the specified members
+    /// of the geospatial index represented by the sorted set at key.
+    ///
+    /// - parameter key: The key of the geospatial index.
+    /// - parameter members: The members from which to get their positions.
+    /// - parameter callback: The callback function.
+    /// - parameter result: An array where each element is a two elements array 
+    ///                     representing longitude and latitude (x,y) of each 
+    ///                     member name passed as argument to the command. Non
+    ///                     existing elements are reported as NULL elements of 
+    ///                     the array.
+    /// - parameter error: Non-nil if error occurred.
+    public func geopos(key: String, members: String..., callback: (_ result: [RedisResponse?]?, _ error: NSError?) -> Void) {
+        geoposArrayOfMembers(key: key, members: members, callback: callback)
+    }
+
+    /// Return the positions (longitude,latitude) of all the specified members
+    /// of the geospatial index represented by the sorted set at key.
+    ///
+    /// - parameter key: The key of the geospatial index.
+    /// - parameter members: The members from which to get their positions.
+    /// - parameter callback: The callback function.
+    /// - parameter result: An array where each element is a two elements array
+    ///                     representing longitude and latitude (x,y) of each
+    ///                     member name passed as argument to the command. Non
+    ///                     existing elements are reported as NULL elements of
+    ///                     the array.
+    /// - parameter error: Non-nil if error occurred.
+    public func geoposArrayOfMembers(key: String, members: [String], callback: (_ result: [RedisResponse?]?, _ error: NSError?) -> Void) {
+        var command = ["GEOPOS", key]
+        for member in members {
+            command.append(member)
+        }
+        issueCommandInArray(command) { (response) in
+            redisArrayResponseHandler(response: response, callback: callback)
+        }
+    }
```

Comment:
Instead of returning `[RedisResponse?]?` Why not return `[(Double, Double)?]?` ? That is an array of optional tuples.